### ISSUE TITLE
Show website icons instead of blank thumbnails

### ISSUE THREAD

**Issue by mzso** [Quicksaver/Tab-Groups#447]
Having a blank thumbnail for unloaded tabs is rather useless.

Also, or alternatively the space could be used to display more of the webpage title.

**Comment by childnode** [comment]
don't get it ... page title is typically shown below and visually there is the favicon shown in the upper thumbnail corner .. what exactly do you see would be helpful here ... more concrete than "more of the webpage title" ...?!

**Comment by mzso** [comment]
[**@childnode**](https://github.com/childnode) commented on [2016. dec. 15. 11:59 CET](https://github.com/Quicksaver/Tab-Groups/issues/447#issuecomment-267300172 "2016-12-15T10:59:45Z - Replied by Github Reply Comments"):
> don't get it, sorry I have to ask ... page title is typically shown below and visually there is the favicon shown in the upper thumbnail corner .. what exactly do you see would be helpful here ... more concrete than "more of the webpage title" ...?! **update** perhaps related to [#352](https://github.com/Quicksaver/Tab-Groups/issues/352) ?

A blank thumbnail is just wasted space...

**Comment by Quicksaver** [comment]
> A blank thumbnail is just wasted space...

Exactly. @childnode I don't understand your comment, what do you believe you gain when you have a blank thumbnail? If there is no information in the thumbnail (it's blank), wouldn't you rather show the information that is available (i.e. the website icon) in a more prominent manner?

**Comment by childnode** [comment]
hi, yes I understand blanc space is wasted space and could be used, but on the other hand: 
- the website icon IS already shown
- the page title IS already shown (or did I missed cases that are adressed here?)
- you might increase it for these cases, but then you break the "visual flow", i.e. it might become to appear cluttering to the user if tabs are rendered differently
- you might duplicate and show it as "website content"
  - which might lead to multiple windows with pixelated icons (most websites do not provide high(er) res favicons)
  - it's even still wasted space with no benefit (if you ask me)

but still, my main point is (despite the fact white, dull space) that I personally can't see which "information" (either grafical or textual) would become a UX benefit

**Comment by Quicksaver** [comment]
> - you might increase it for these cases, but then you break the "visual flow", i.e. it might become to appear cluttering to the user if tabs are rendered differently
> - you might duplicate and show it as "website content"
>   - which might lead to multiple windows with pixelated icons (most websites do not provide high(er) res favicons)
>   - it's even still wasted space with no benefit (if you ask me)

What is "it"? The icon wouldn't be increased at all, it would be rendered pretty much as is happening now when thumbnails become too small and the groups tiles only the icons instead. Except with this suggestion it would be a mix of both thumbnails and tiled icons within a group.

> but still, my main point is (despite the fact white, dull space) that I personally can't see which "information" (either grafical or textual) would become a UX benefit

The benefit would come from having less "blank squares" clogging up the UI, essentially cleaning it from "useless information". In other words, it's better to have an "emptier interface" (i.e. smaller icons tiled, with larger margins between them and the tab items limits) than an "interface full of empty information" (i.e. larger thumbnails closer to the tab items margins but that don't carry any visible info because they're blank).

> if you add more textual information for these 'blanc preview' cases, what about those who then want these information for websites with preview too?

FYI that probably won't happen, just the tiling icons part because that's what makes more sense to keep the interface consistent. But to answer your question, you can always disable thumbnails entirely and the tabs will be shown in a list where you can see their full title and even urls (provided they fit on screen of course).

**Comment by childnode** [comment]
Hi again @Quicksaver ,

> What is "it"?

the favicon, but you grabbed correctly it in your paragraph ;)

> The benefit would come from having less "blank squares", essentially cleaning it from "useless information". In other words, it's better to have an "emptier interface"

Sorry, I can't follow this \<P>. Didn't I say "duplicate information is bad, better less than ..." and you're talking about the same?!
Perhaps we are talking about the same, perhaps not ... I don't know, but I see, you might have another point of view to this, that's ok for me. I just wanted to throw my thoughts in. Perhaps we just should go forward for a mockup to "see" what happens ;)

With kind regards,
~Marcel
